platform-includes/getting-started-sourcemaps/javascript.react-router.mdx

```diff
@@ -1,9 +1,43 @@
-## Add Readable Stack Traces to Errors
+## Source Maps Upload
 
-By default, React Router will minify your JavaScript and CSS files in production. This makes it difficult to debug errors. To make debugging easier, you can generate source maps and upload them to Sentry.
+Update `vite.config.ts` to include the `sentryReactRouter` plugin and also add your `SentryReactRouterBuildOptions` config options to the vite config (this is required for uploading source maps at the end of the build):
 
-We recommend using [Sentry's Vite plugin](/platforms/javascript/sourcemaps/uploading/vite/) to upload sourcemaps.
+<OrgAuthTokenNote />
 
-Please refer to the <PlatformLink to="/sourcemaps">Source Maps Documentation</PlatformLink>, for more information.
+```typescript {filename: vite.config.ts}
+import { reactRouter } from '@react-router/dev/vite';
+import { sentryReactRouter, type SentryReactRouterBuildOptions } from '@sentry/react-router';
+import { defineConfig } from 'vite';
 
-For more advanced configuration, you can use [`sentry-cli`](https://github.com/getsentry/sentry-cli) directly to upload source maps.
+const sentryConfig: SentryReactRouterBuildOptions = {
+  org: "___ORG_SLUG___",
+  project: "___PROJECT_SLUG___",
+
+  // An auth token is required for uploading source maps.
+  authToken: "___ORG_AUTH_TOKEN___"
+  // ...
+};
+
+export default defineConfig(config => {
+  return {
+    plugins: [reactRouter(), sentryReactRouter(sentryConfig, config)],
+    sentryConfig,
+  };
+});
+```
+
+Include the `sentryOnBuildEnd` hook in `react-router.config.ts`:
+
+```typescript {filename: react-router.config.ts}
```

Comment:
You could highlight this line
```suggestion
```typescript {filename: react-router.config.ts} {9}
```